addons/project_mail_plugin/controllers/project_client.py

```diff
@@ -0,0 +1,41 @@
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import html2plaintext
+
+
+class ProjectClient(http.Controller):
+    @http.route('/mail_plugin/project/search', type='json', auth='outlook', cors="*")
+    def projects_search(self, search_term, limit=5):
+
+        projects = request.env['project.project'].search([('name', 'ilike', search_term)], limit=limit)
+
+        return [
+            {
+                'name': project.name,
+                'project_id': project.id,
```

Comment:
When searching task, you put 'task_id' as the first key, maybe do the same here by putting 'project_id' as the first key?
(Yes, I like consistency :smile: )

Reply:
Done, 'project_id' is now the first key :)